js/src/logger.ts

```diff
@@ -202,6 +218,23 @@ class BraintrustState {
     }
     return this._logConn!;
   }
+
+  // Note: the format should be the same as in logger.py.
+  public serializeLoginInfo(): string {
+    return JSON.stringify({
+      api_url: this.apiUrl,
+      login_token: this.loginToken,
+      org_id: this.orgId,
+      org_name: this.orgName,
+      log_url: this.logUrl,
+    });
+  }
```

Comment:
I think this is redundant, and if you use your `BRAINTRUST_API_KEY` we already re-derive this info in the logger. Perhaps we should just send `x-bt-org-name`, and let the logic in the proxy handle the rest?